ledger-core-ethereum/test/ExplorerTests.cpp

```diff
@@ -54,4 +55,16 @@ TEST_F(EthereumExplorers, GetGasPrice) {
 TEST_F(EthereumExplorers, GetEstimatedGasLimit) {
     auto result = wait(explorer->getEstimatedGasLimit("0x57e8ba2a915285f984988282ab9346c1336a4e11"));
     EXPECT_GE(result->toUint64(), 10000);
+}
+
+TEST_F(EthereumExplorers, PostEstimatedGasLimit) {
+  auto request = api::EthereumGasLimitRequest(
+      optional<std::string>(), optional<std::string>(), optional<std::string>(),
+      optional<std::string>("0xa9059cbb00000000000000000000000013C5d95f25688f8A"
```

Comment:
You don’t need the `optional` here, as mentioned in the original PR.

Reply:
I added `optional` because an error occured when compiling with apple-clang 11.0

![Screenshot 2020-03-17 at 10 08 15](https://user-images.githubusercontent.com/6042495/76840282-3f237600-6837-11ea-9367-457c1712fdfd.png)

Reply:
[Yes, I already fixed that lately](https://github.com/LedgerHQ/lib-ledger-core/pull/514/files).